config/software/postgresql.rb

```diff
@@ -28,78 +28,33 @@
 dependency "libossp-uuid"
 dependency "config_guess"
 
-#
+# version_list: url=https://ftp.postgresql.org/pub/source/v#{version}/ filter=*.tar.bz2
+
+version("13.3")   { source sha256: "3cd9454fa8c7a6255b6743b767700925ead1b9ab0d7a0f9dcb1151010f8eb4a1" }
+
 # Version 12.x will EoL November 14, 2024
-#
-version "12.5" do
-  source sha256: "bd0d25341d9578b5473c9506300022de26370879581f5fddd243a886ce79ff95",
-  url: "https://ftp.postgresql.org/pub/source/v12.5/postgresql-12.5.tar.bz2"
-end
+version("12.5")   { source sha256: "bd0d25341d9578b5473c9506300022de26370879581f5fddd243a886ce79ff95" }
 
-#
 # Version 9.6 will EoL November 11, 2021
-#
-version "9.6.10" do
-  source sha256: "8615acc56646401f0ede97a767dfd27ce07a8ae9c952afdb57163b7234fe8426"
-end
+version("9.6.10") { source sha256: "8615acc56646401f0ede97a767dfd27ce07a8ae9c952afdb57163b7234fe8426" }
 
-#
 # Verison 9.5 was EoL February 11, 2021
-#
-version "9.5.14" do
-  source sha256: "3e2cd5ea0117431f72c9917c1bbad578ea68732cb284d1691f37356ca0301a4d"
-end
+version("9.5.14") { source sha256: "3e2cd5ea0117431f72c9917c1bbad578ea68732cb284d1691f37356ca0301a4d" }
```

Comment:
Assuming we're not using this anywhere we should remove it

Reply:
Removed it